test/test-main.js

```diff
@@ -0,0 +1,10 @@
+global.DATABASE_URL = 'mongodb://localhost/basic-auth-test'
+global.PORT = '8081'
```

Comment:
FYI/FWIW, that change in the restaurants app didn't make it's way into basic-auth app yet, so there was no way for you to know.